extensions/hibernate-validator/deployment/src/main/java/io/quarkus/hibernate/validator/deployment/HibernateValidatorProcessor.java

```diff
@@ -159,7 +161,15 @@ void registerAdditionalBeans(HibernateValidatorRecorder hibernateValidatorRecord
             additionalBeans.produce(new AdditionalBeanBuildItem(
                     "io.quarkus.hibernate.validator.runtime.jaxrs.ResteasyReactiveEndPointValidationInterceptor"));
             additionalBeans.produce(new AdditionalBeanBuildItem(
-                    "io.quarkus.hibernate.validator.runtime.jaxrs.ResteasyReactiveContextLocaleResolver"));
+                    "io.quarkus.hibernate.validator.runtime.locale.LocaleResolversWrapper"));
+            additionalBeans.produce(new AdditionalBeanBuildItem(
+                    "io.quarkus.hibernate.validator.runtime.locale.ResteasyReactiveContextLocaleResolver"));
+        }
+        if (capabilities.isPresent(Capability.SMALLRYE_GRAPHQL)) {
```

Comment:
Question: What happens in the case where both RESTEasy Reactive and GraphQL are being used?

Reply:
The idea is that both resolvers will be invoked, but only one of them should actually return a locale, the other one should return null, depending on whether the request is for JAX-RS or GraphQL. 

I now realize though that the `VertxLocaleResolver` (meant to be used by GraphQL) will probably return a locale even for a JAX-RS request, because it looks into the `CurrentVertxRequest`. Given that (I think) RR does not support Undertow and there will always be a Vert.x request, perhaps we could remove the `ResteasyReactiveContextLocaleResolver` and use `VertxLocaleResolver` for that case?

For resteasy classic I think we have to keep `ResteasyContextLocaleResolver` because there might be Undertow underneath

Reply:
Yeah I tried that and it worked, I think I'll remove the `ResteasyReactiveContextLocaleResolver` and just use the vertx one for resteasy-reactive

Reply:
The Vert.x resolver is handling things at the HTTP level in a similar way? I wouldn't want things to become inconsistent between HV and the REST request.

Reply:
Yeah it behaves the same, we just read headers directly from the Vert.x request instead of retrieving an instance  of `javax.ws.rs.core.HttpHeaders` from some resteasy context.